docs/rfcs/0001-rfc-process.md

```diff
@@ -0,0 +1,63 @@
+- Status: Proposed
```

Comment:
If the PR is active, I imagine we can assume it is in the "proposed" status. Once the reviewers approve, it'll become approved but then the author has to go back and create another commit to change this status. Does it make more sense to have the status in the body of the PR so it's easy to change?

Reply:
You can change with a suggestion like:
```suggestion
- Status: Accepted
```
Which is easy enough IMO. There is always the possibility of an RFC being declined... Which would be nice to see right as you open it. 

One thing though, we should not run CI for PRs with the RFC prefix. Is that possible? cc @strseb We probably still want to run linters for spellcheck, but othewise it's all unnecessary.

Reply:
Good callout. So we can stop the ci to run - however not by PR name but for example for pr's on a `rfc/*` branch. 
But we cannot remove the branch protection rules per PR. 
Meaning: 
- Only gh admins can merge those.
- We need a diffrent repo to store them.

Reply:
Got it! I mean... We could always make every engineer in the team a GH admin @lesleyjanenorton 👀

Reply:
We don't necessarily need an explicit status in the RFC. It could be implicit which is how we use it on Android: 
- Opening the PR is PROPOSED 
- Merging the PR is ACCEPTED
- Closing the PR (unmerged, but with summary of outcome) is DECLINED

However, if we prefer having the DECLINED RFCs merged for the purpose of documentation and having it all in one place, I think that's fine too! We just have to remember to update the status before merging.

Reply:
I see value in merging declined RFCs, FWIW.